modules/control/controller/lon_controller.cc

```diff
@@ -295,15 +299,17 @@ Status LonController::ComputeControlCommand(
   }
 
   if (calibration_value >= 0) {
-    throttle_cmd = std::abs(calibration_value) > throttle_deadzone
+    throttle_cmd = std::abs(calibration_value) >
+                           std::max(throttle_deadzone, throttle_minimum_action)
                        ? std::abs(calibration_value)
-                       : throttle_deadzone;
+                       : std::max(throttle_deadzone, throttle_minimum_action);
```

Comment:
Plz check out the `DRY principle`: https://github.com/ApolloAuto/apollo/pull/7514/files

Reply:
Clean the codes according to the Dry Principle. Thanks.